common/proto/proto.go

```diff
@@ -0,0 +1,21 @@
+// Library for common gRPC protobuf-related tools.
+
```

Comment:
Think the pkg level comments only work if there isn't a blank line between comment & pkg name